### PR TITLE
Santosh Kumar Gajawada: UnExport GCS util fuctions

### LINKED ISSUE #22: Make integration and acceptance tests are under one single package
So that all the GCS util functions can be unexported

### PR DESCRIPTION
**Pull request by WeirdMagician**
Fixes #22 

### Quick checks:

- [ ] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [ ] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-s3/pulls) for the same update/change.
- [ ] I have written unit tests.
- [ ] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: GCS_util.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package source
+package connector
```
Comment (hariso) [inline GCS_util.go:15]: Why not call it source_test?

Comment (hariso) [inline GCS_util.go:15]: Why not call it source_test?

Comment (WeirdMagician) [reply GCS_util.go:15]: I think we have one directory and one package in Golang.
In the parent directory we are using connector package so iam using the same for util and test files too.

Comment (WeirdMagician) [reply GCS_util.go:15]: I think we have one directory and one package in Golang.
In the parent directory we are using connector package so iam using the same for util and test files too.

**Review by hariso — APPROVED** [review]

**Pull request merged**